Loads/Loads/Point2dDisplacement.cs

```diff
@@ -0,0 +1,18 @@
+using OasysUnits;
+
+namespace MagmaWorks.Taxonomy.Loads
+{
+    public class Point2dDisplacement : IPoint2dDisplacement
+    {
+        public Length X { get; set; } = Length.Zero;
+        public Length Z { get; set; } = Length.Zero;
+
+        internal Point2dDisplacement() { }
+
+        public Point2dDisplacement(Length x, Length z)
+        {
+            X = x;
+            Z = z;
+        }
+    }
```

Comment:
No operator here?

Reply:
Just wanted to add a few commonly used casts (from Force to Load seemed reasonable) but not for everything